### DISCUSSION TITLE
doubled AreaPostprocessor results

### DISCUSSION THREAD

**Discussion by Joseph-0123** [idaholab/moose#25544] [Q&A Meshing] [answered]
Dear all, 

I use AreaPostprocessor to set the proper 'flux_function' for PorousFlowSink BC. 

The model contains a 50m(X)*5m (Y)*50m (Z) reservoir within the 100m*100m*100m cubic.

I set the PorousFlowSink BCs on both the inj_well (in green, 5m*50m=250 m2) and pro_well (in blue, 5m*50m=250 m2).

But the output of AreaPostprocessor is 500 m2, two times of actual area. But the volume of the reservoir is 125000 m3, and the area results of top and bottom boundaries are 1000 m2. These are correct.

Could you please help me with this?

**Comment by GiudGiud** [comment, marked as answer]
Hello

this is a meshing issue most likely
Either:
- two layers of sides are included in the sideset you re measuring the area of
- sides in both directions (normal pointing both ways) are included in the sideset

both are problematic, and will force you to redo your mesh generation
You can also work with a ParsedGenerateSideset to restrict the sideset to what you want. Maybe indicating the included_sideset and included subdomains will be enough to restrict the sideset

Guillaume

**Reply by Joseph-0123** [reply]
Hi Guillaume,

thanks a lot! I use ParsedGenerateSideset and get the correct area of 250 m2.

Best regards,

J
